docs/source/tracking/tutorials/local-database.rst

```diff
@@ -0,0 +1,103 @@
+========================================
+Tracking Experiments with Local Database
+========================================
+
+In this tutorial, you will learn how to use local database to track your experiment metadata with MLflow. By default, MLflow Tracking logs run to local files,
+which may cause some frustration due to fractured small files and lack of simple access interface. Also, if you are using Python, you can use SQLite that runs 
+upon local file (e.g. ``mlruns.db``) and has a built-in client ``sqlite3``, eliminating the effort to install any additional dependencies and setting up database server.
+
+Get Started
+===========
+
+Step 1 - Get MLflow
+-------------------
+MLflow is available on PyPI. If you don't already have it installed on your local machine, you can install it with:
+
+.. code-section::
+
+    .. code-block:: bash
+        :name: download-mlflow
+
+        pip install mlflow
+
+Step 2 - Configure MLflow environment varialbles
+------------------------------------------------
+
+Set tracking URI to local SQLite database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To point MLflow to your local SQLite database, you need to set envioronment variable ``MLFLOW_TRACKING_URI`` to ``sqlite:///mlruns.db``.
+(This will create a SQLite database file called ``mlruns.db`` in the current directory. Specify different path if you want to store the database file in a different location.)
```

Comment:
```suggestion
(This will create a SQLite database file called ``mlruns.db`` in the current directory. Specify a different path if you want to store the database file in a different location.)
```